source/agora/common/crypto/Key.d

```diff
@@ -322,7 +321,7 @@ public struct Seed
     }
 
     /// Make sure the sink overload of BitBlob is not picked
-    public void toString (scope void delegate(const(char)[]) sink) const
+    public void toString (scope void delegate(const(char)[]) @safe sink) const
```

Comment:
If you force `@safe` on the delegate, you should force `@safe` on `toString`.

Reply:
Is it ok if `toString` for `Seed` is not `@safe`?
It's using `Base32.encode`, which is not safe.

Reply:
Then why did you change the delegate to be `@safe` ?

Reply:
I was assuming it's possible for only one of them to be `@safe`.. 😅